wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -231,21 +231,11 @@ The number of entries to display for each page. The minimum is **1**, whereas th
 is **$maxValue**. If nothing is specified, **this value defaults to $defaultValue**.
 |]
 
-instance ToParamSchema PerPage where
-  toParamSchema _ = mempty
-    & type_ .~ SwaggerInteger
-    & default_ ?~ (Number $ fromIntegral defaultPerPageEntries)
-    & minimum_ ?~ 1
-    & maximum_ ?~ (fromIntegral maxPerPageEntries)
-
-instance ToParamSchema Page where
-  toParamSchema _ = mempty
-    & type_ .~ SwaggerInteger
-    & default_ ?~ (Number 1) -- Always show the first page by default.
-    & minimum_ ?~ 1
```

Comment:
I might be knocking out a Schelling fence here, but I can't see a reason for these to be orphaned when the library is in the same package.